testdata/test_types_extra/types.go

```diff
@@ -0,0 +1,12 @@
+package test_types_extra
+
+import "github.com/wI2L/fizz/testdata/test_types"
+
+type W struct {
+	C, D int
+}
+
+type D struct {
+	W  W
```

Comment:
Just a nitpick, but to convey the intent, I'd rename the fields `Winternal`, and `Wexternal`, or something like to indicate that.